### ISSUE TITLE
Future warning: Passing a set as indexer - SmartCorrelatedSelection

### ISSUE THREAD

**Issue by giemmecci** [feature-engine/feature_engine#453]
**Describe the bug**
Hi, I'm getting a FutureWarning message when running the SmartCorrelatedSelection both on my data and following one of the examples from the tutorial:

```
/redacted_path/lib/python3.9/site-packages/feature_engine/selection/smart_correlation_selection.py:302: FutureWarning: Passing a set as an indexer is deprecated and will raise in a future version. Use a list instead.
  f = X[feature_group].std().sort_values(ascending=False).index[0]
/redacted_path/lib/python3.9/site-packages/feature_engine/selection/smart_correlation_selection.py:302: FutureWarning: Passing a set as an indexer is deprecated and will raise in a future version. Use a list instead.
  f = X[feature_group].std().sort_values(ascending=False).index[0]
```

**To Reproduce**
I have followed the code example posted [here](https://feature-engine.readthedocs.io/en/1.3.x/user_guide/selection/SmartCorrelatedSelection.html)

**Screenshots**

![Screenshot from 2022-05-11 12-23-26](https://user-images.githubusercontent.com/37995198/167911651-0b9eb76b-85f4-43ca-a232-d2bc2c512dd2.png)

**Desktop:**
 - OS: Centos 7
 - Browser: Chrome
 - Feature_engine Version: 1.3.0

Thanks

**Comment by solegalli** [comment]
Hi @giemmecci 

thank you for the raising the issue.

Can I ask which version of pandas you are using?

**Comment by solegalli** [comment]
fyi

#455

**Comment by solegalli** [comment]
resolved in #455
